apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -2,21 +2,35 @@
   <LayoutModal :title="title" :show="isModalShown" @close="handleClose">
     <template #body>
       <div class="d-flex">
-        <EditModalWizard
-          v-if="loaded && tableMetaData"
-          :id="id"
-          v-model="rowData"
-          :pkey="pkey"
-          :tableName="tableName"
-          :tableMetaData="tableMetaData"
-          :graphqlURL="graphqlURL"
-          :visibleColumns="visibleColumns"
-          :clone="clone"
-          :page="currentPage"
-          @setPageCount="pageCount = $event"
-          class="flex-grow-1"
-        >
-        </EditModalWizard>
+        <template v-if="useChapters">
```

Comment:
What is the advantage of using template here instead of just a div?

Reply:
no to add the extra div , the template allows for controle logic ( use comp A or Comp B ) without changing the dom tree